pdpipe/core.py

```diff
@@ -191,6 +193,7 @@ def __init__(self, exraise=True, exmsg=None, desc=None, prec=None,
         self._prec_arg = prec
         self._skip = skip
         self._appmsg = '{}..'.format(desc)
+        self._label = label
```

Comment:
I would check `label` and raise error if (at least) not a str

Reply:
Agreed. Use `with pytest.raises(SomeException)` to check for correct error throwing behaviour.

Reply:
```python 
def __init__(self, label='', ...)
    if not isinstance(label, str):
        raise Exception(f"label must be a str, not {type(label)}.")
    self._label = label
    ...
```